web/src/main/kotlin/no/nav/su/se/bakover/web/routes/dokument/DokumentJson.kt

```diff
@@ -13,6 +13,8 @@ internal fun Dokument.toJson(): DokumentJson {
         tittel = tittel,
         opprettet = DateTimeFormatter.ISO_INSTANT.format(opprettet),
         dokument = generertDokument,
+        journalført = (this is Dokument.MedMetadata && metadata.journalpostId.isNotNullOrEmpty()),
```

Comment:
Har det vist seg å være et problem med tom journalpostId/brevbestillingId? Da burde vi kanskje legge inn en validering eller sanering i ctor/tryCreate i respektive klasser? `JournalpostId` og `BrevbestillingId. Og gjøre en migrering.

Reply:
Nei, ikke relatert til dette. Her i DTOen er dette bare en sjekk for å få en boolean.

Men [her](https://github.com/navikt/su-se-bakover/pull/543/files/2a69ecd1fb388413c73bdeaa8174bf3277f97a98#diff-d0f057d9812ca994a2820f84788b38abae1d0b647aeafe7ca4c57bd5c10f7c28R232) må vi sjekke om vi skal hente feltene fra db, ettersom det krasjer hvis kolonen ikke er hentet fra sql-spørringa

Reply:
Jeg refererte til at vi bruker nullOrEmpty. De IDene bør aldri være en tom streng, verken i domenet eller basen 😬

Reply:
De skal heller aldri være en tom streng, er bare en litt mer dekkende sjekk. Hvis vi kan garantere at de aldri er en tom streng i db, så kan jeg godt bytte :)